src/catalog/catalog_entry/sequence_catalog_entry.cpp

```diff
@@ -96,7 +146,7 @@ unique_ptr<CreateInfo> SequenceCatalogEntry::GetInfo() const {
 	result->increment = seq_data.increment;
 	result->min_value = seq_data.min_value;
 	result->max_value = seq_data.max_value;
-	result->start_value = seq_data.counter;
+	result->start_value = seq_data.start_value;
```

Comment:
Let's try to avoid changing the sequence serialization here as this might influence backwards compatibility